cdc/model/sink_test.go

```diff
@@ -396,3 +398,168 @@ func TestExchangeTablePartition(t *testing.T) {
 	require.Equal(t, "ALTER TABLE `test1`.`t1` EXCHANGE PARTITION `p0` WITH TABLE `test2`.`t2`", event.Query)
 	require.Equal(t, event.Type, timodel.ActionExchangeTablePartition)
 }
+
+func TestSortRowChangedEvent(t *testing.T) {
+	events := []*RowChangedEvent{
+		{
+			PreColumns: []*Column{{}},
+			Columns:    []*Column{{}},
+		},
+		{
+			Columns: []*Column{{}},
+		},
+		{
+			PreColumns: []*Column{{}},
+		},
+	}
+	assert.True(t, events[0].IsUpdate())
+	assert.True(t, events[1].IsInsert())
+	assert.True(t, events[2].IsDelete())
+	sort.Sort(txnRows(events))
```

Comment:
I think the sort functionality is covered by other case, no need to have this case.

If put the sort into the `trySplitAndSortUpdateEvent`, only test this method with multiple difference cases should be ok.

Reply:
yes, but let keep this test case to just the test order b logic